frontend/public/module/k8s/service-catalog.ts

```diff
@@ -7,6 +7,9 @@ import { K8sResourceKind } from '../../module/k8s';
 export const serviceClassDisplayName = (serviceClass: K8sResourceKind): string =>
   _.get(serviceClass, 'spec.externalMetadata.displayName') || _.get(serviceClass, 'spec.externalName');
 
+export const servicePlanDisplayName = (servicePlan: K8sResourceKind): string =>
+  _.get(servicePlan, 'spec.externalName') || servicePlan.metadata.name;
```

Comment:
I believe plans have a display name, too

Reply:
I couldn't find any reference in that types.go doc, but copied what was defined for serviceclass.

Reply:
`externalMetadata` can be anything and is completely up to the broker, but there are some common conventions for service classes and plans including display name:

https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md#service-metadata